contents/handbook/engineering/support-hero.md

```diff
@@ -53,52 +77,20 @@ As a business we need to ensure we are focusing support on our paying customers,
 1. [Community questions](https://posthog.com/questions/)
```

Comment:
```suggestion
```

Community questions will be correctly tagged in Zendesk already. Additionally, it doesn't make sense to look at posthog.com/questions since you'll encounter a bunch of tickets that are not related to your team anyway.

Reply:
Good point, removed